packages/wdio-browserstack-service/tests/service.test.js

```diff
@@ -184,16 +184,57 @@ describe('afterSuite', () => {
 describe('afterTest', () => {
     it('should increment failures on fails', () => {
         service.failures = 0
+        service.fullTitle = ''
+        service.failReason = ''
 
-        service.afterTest({ passed: false })
+        service.afterTest({ passed: false, parent: 'foo', title: 'bar', error: {message: 'error message'}  })
         expect(service.failures).toBe(1)
+        expect(service.fullTitle).toBe('foo - bar')
+        expect(service.failReason).toBe('error message')
     })
 
     it('should not increment failures on passes', () => {
         service.failures = 0
+        service.fullTitle = ''
+        service.failReason = ''
 
-        service.afterTest({ passed: true })
+        service.afterTest({ passed: true, parent: 'foo', title: 'bar' })
         expect(service.failures).toBe(0)
+        expect(service.fullTitle).toBe('foo - bar')
+    })
+
+    it('should not increment failures on passes', () => {
+        service.failures = 0
+        service.fullTitle = ''
+        service.failReason = ''
+
+        service.afterTest({ passed: true, parent: 'foo', title: 'bar' })
+        expect(service.failures).toBe(0)
+    })
+
+    it('should add test name parameter on pases', () => {
+        service.failures = 0
+        service.fullTitle = ''
+        service.failReason = ''
+
+        service.afterTest({ passed: true, parent: 'foo', title: 'bar' })
+        expect(service.fullTitle).toBe('foo - bar')
+    })
+
+    it('should contain or not reason value on pases or fails', () => {
+        service.failures = 0
+        service.fullTitle = ''
+        service.failReason = ''
+
+        service.afterTest({ passed: true, parent: 'foo', title: 'bar' })
```

Comment:
this tests the same again as `should not increment failures on passes` and can be removed